tsdb/engine/tsm1/engine.go

```diff
@@ -1766,19 +1770,20 @@ func (e *Engine) deleteSeriesRange(seriesKeys [][]byte, min, max int64) error {
 			return err
 		}
 
-		fielsetChanged := false
+		actuallyDeleted := make([]string, 0, len(measurements))
 		for k := range measurements {
 			if dropped, err := e.index.DropMeasurementIfSeriesNotExist([]byte(k)); err != nil {
 				return err
```

Comment:
If we return here because `DropMeasurementIfSeriesNotExist` fails, do we need to anything if `actuallyDeleted` is not empty?

Reply:
https://github.com/influxdata/influxdb/pull/23701#discussion_r961182146